src/services/sdkService.ts

```diff
@@ -98,8 +99,8 @@ export default class SdkService {
         ...(gasSpeed ? { gasSpeed: { speed: gasSpeed, requirement: 'best effort' } } : {}),
         ...(skipValidation ? { skipValidation } : {}),
         ...(isBuyOrder ? { estimateBuyOrdersWithSellOnlySources: true } : {}),
-        quoteTimeout: '5s',
```

Comment:
Moved the timeout out of the request and onto the config, to make it consistent with all other services